.NET/Microsoft.Recognizers.Text.Sequence/Chinese/Extractors/ChineseCreditCardExtractorConfiguration.cs

```diff
@@ -0,0 +1,31 @@
+﻿using System.Collections.Generic;
+using System.Collections.Immutable;
+using System.Text.RegularExpressions;
+using Microsoft.Recognizers.Definitions.Chinese;
+
+namespace Microsoft.Recognizers.Text.Sequence.Chinese
+{
+    public class ChineseCreditCardExtractorConfiguration : CreditCardConfiguration
+    {
+
+        public ChineseCreditCardExtractorConfiguration(SequenceOptions options)
+            : base(options)
+        {
+            var regexes = new Dictionary<Regex, string>
+            {
+                {
+                    new Regex(CreditCardDefinitions.ChinaUnionPayRegex, RegexOptions.Compiled),
```

Comment:
This is very inefficient just to check for prefixes. And it would get worst as more and more options are added here.